projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-utils.php

```diff
@@ -23,11 +23,9 @@ class Jetpack_SEO_Utils {
 	/**
 	 * Used to check whether SEO tools are enabled for given site.
 	 *
-	 * @param int $site_id Optional. Defaults to current blog id if not given.
-	 *
 	 * @return bool True if SEO tools are enabled, false otherwise.
 	 */
-	public static function is_enabled_jetpack_seo( $site_id = 0 ) {
+	public static function is_enabled_jetpack_seo() {
```

Comment:
Technically I guess you could say this is an API change, but I confirmed nothing in WPCOM is calling it with a site ID passed in, and just in case something else out there is, it wouldn't actually change the behavior anyway (because the function always returns `true` for Jetpack).

So :+1: from me at least.